iOS/MobileStarter/react-app/public/locales/en/ReactApp.json

```diff
@@ -24,20 +24,24 @@
   },
   "HubScreen": {
     "Connecting": "Connecting...",
-    "GettingProjectList": "Getting Project List...",
     "SelectProject": "Select Project",
-    "GettingIModelList": "Getting iModel List...",
-    "DownloadingIModel": "Downloading iModel...",
+    "DownloadingIModel": "Downloading iModel",
     "SignOut": "Sign Out",
     "FetchProjectsErrorFormat": "Error getting projects list: {{- error}}",
     "GetIModelsErrorFormat": "Error getting iModels list: {{- error}}",
     "SigninErrorFormat": "Error signing in: {{- error}}",
     "DeleteAllDownloads": "Delete All Downloads",
     "ChangeProject": "Change Project",
-    "Filter": "Filter",
+    "Search": "Search",
     "DownloadErrorFormat": "Error downloading iModel: {{- error}}",
     "DeleteErrorFormat": "Error deleting local download of iModel: {{- error}}",
-    "DeleteAllErrorFormat": "Error deleting local downloads of all iModels in project: {{- error}}"
+    "DeleteAllErrorFormat": "Error deleting local downloads of all iModels in project: {{- error}}",
+    "All": "ALL",
+    "Recents": "RECENT",
+    "Favorites": "FAVORITES",
```

Comment:
First of all, I don't feel that these look good in all caps. Having said that, if all caps is desired, then I feel that that should be enforced by the CSS (like the bottom tabs in the model UI), not the i18n text.

Secondly, with the current font size and all caps, "FAVORITES" is cramped on a minimum size phone screen.

Reply:
Changed to mixed case.